### PR TITLE
Added more performance counters to SqlServer input plugin

### PR DESCRIPTION
**Pull request by avinash-nigam**
### Required for all PRs:

- [x] Signed [CLA](https://influxdata.com/community/cla/).
- [ ] Associated README.md updated.
- [ ] Has appropriate unit tests.

### CODE REVIEW

**Comment by avinash-nigam** [comment]
@denzilribeiro Could you please review this PR for adding more perf counters to the SqlServer plugin?

**Review by denzilribeiro — APPROVED** [review]
Looks good - you think "Logical connections" isn't an edge case though?

**Comment by avinash-nigam** [comment]
> Looks good - you think "Logical connections" isn't an edge case though?

We did few customers asking for the same, hence including the same

**Comment by avinash-nigam** [comment]
@ssoroka and @reimda - could you please help merge this pull request?

**Comment by reimda** [comment]
> @ssoroka and @reimda - could you please help merge this pull request?

Could you include a link to documentation where these counter names are described, either from Microsoft or go-mssqldb?

**Comment by avinash-nigam** [comment]
> > @ssoroka and @reimda - could you please help merge this pull request?
> 
> Could you include a link to documentation where these counter names are described, either from Microsoft or go-mssqldb?

Yes, here are the Microsoft documentation links for each counter added - 

Table Lock Escalations/sec - [MSSQL$*:Access Methods](https://docs.microsoft.com/en-us/sql/relational-databases/performance-monitor/sql-server-access-methods-object?view=sql-server-ver15)
Free pages - [MSSQL$*:Buffer Manager](https://docs.microsoft.com/en-us/sql/relational-databases/performance-monitor/sql-server-buffer-manager-object?view=sql-server-ver15)
Active Transactions - [MSSQL$*:Databases](https://docs.microsoft.com/en-us/sql/relational-databases/performance-monitor/sql-server-databases-object?view=sql-server-ver15)
Log Growths - [MSSQL$*:Databases](https://docs.microsoft.com/en-us/sql/relational-databases/performance-monitor/sql-server-databases-object?view=sql-server-ver15)
Logical Connections - [MSSQL$*:General Statistics](https://docs.microsoft.com/en-us/sql/relational-databases/performance-monitor/sql-server-general-statistics-object?view=sql-server-ver15)
Distributed Query - MSSQL$*:[Exec Statistics](https://docs.microsoft.com/en-us/sql/relational-databases/performance-monitor/sql-server-execstatistics-object?view=sql-server-ver15)
DTC calls - MSSQL$*:[Exec Statistics](https://docs.microsoft.com/en-us/sql/relational-databases/performance-monitor/sql-server-execstatistics-object?view=sql-server-ver15)
Query Store CPU usage - [SQLServer:Query Store](https://docs.microsoft.com/en-us/sql/relational-databases/performance-monitor/sql-server-query-store-object?view=sql-server-ver15)

**Comment by avinash-nigam** [comment]
@reimda - Please let me know if you need further information on these counters before merging the pull request. Thank you!

File: plugins/inputs/sqlserver/sqlqueriesV2.go
Comment (reimda) [inline plugins/inputs/sqlserver/sqlqueriesV2.go:468]: Could you put those documentation links for the counters in the source code, maybe on the line before declaring sqlPerformanceCountersV2?  I'm hoping doc links will help out future maintainers who are not as familiar with SQL Server.  Thanks!

**Review by reimda — CHANGES_REQUESTED** [review]

**Review by reimda — APPROVED** [review]

File: plugins/inputs/sqlserver/sqlqueriesV2.go
Comment (avinash-nigam) [reply plugins/inputs/sqlserver/sqlqueriesV2.go:479]: > Could you put those documentation links for the counters in the source code, maybe on the line before declaring sqlPerformanceCountersV2? I'm hoping doc links will help out future maintainers who are not as familiar with SQL Server. Thanks!

@reimda Thank you for the prompt review of the changes

**Pull request merged**
